src/components/Layout.tsx

```diff
@@ -1,18 +1,39 @@
 
-import Nav from './Nav'
+import Header from './Header'
+import Footer from './Footer'
+import { Box } from '@mui/material'
+import { useRouter } from 'next/router'
 
 interface Props {
     children: any
 }
 
 const Layout = ({children}: Props) => {
-    return (
-        <>
-            <Nav/>
-            <div>
-                {children}
-            </div>
-        </>
-    )
+    const router = useRouter();
```

Comment:
```suggestion
    const { route } = useRouter();
```
바로 디스트럭쳐링 및 `pathname`으로 비교하면 될 것 같아요
[router-object](https://nextjs.org/docs/api-reference/next/router#router-object)